src/processTargets/modifiers/ModifyIfUntypedStage.ts

```diff
@@ -1,38 +1,77 @@
 import { Target } from "../../typings/target.types";
-import { ModifyIfUntypedModifier } from "../../typings/targetDescriptor.types";
+import {
+  Modifier,
+  ModifyIfUntypedModifier,
+} from "../../typings/targetDescriptor.types";
 import { ProcessedTargetsContext } from "../../typings/Types";
 import getModifierStage from "../getModifierStage";
 import { ModifierStage } from "../PipelineStages.types";
 
-/**
- * Runs {@link ModifyIfUntypedModifier.modifier} if the target has no explicit
- * scope type, ie if {@link Target.hasExplicitScopeType} is `false`.
- */
-export default class ModifyIfUntypedStage implements ModifierStage {
+abstract class ModifyIfBaseStage implements ModifierStage {
   private nestedStage_?: ModifierStage;
 
-  constructor(private modifier: ModifyIfUntypedModifier) {}
+  constructor(private modifier: Modifier, private suppressErrors?: boolean) {}
```

Comment:
Also, I think I'd call the nested modifier `nestedModifier`, because usually `modifier` refers to the modifier for which we are constructing a stage